src/libs/konnectors/sendKonnectorsLogs.spec.js

```diff
@@ -176,4 +176,126 @@ describe('Konnectors logs', () => {
       type: 'konnectorLogs/removeLogs'
     })
   })
+
+  it('Should send logs for 2 job ids and without', async () => {
+    const client = new CozyClient()
+
+    store.getState.mockReturnValue({
+      konnectorLogs: {
+        logs: {
+          myslug: [
+            {
+              jobId: 'jobIdA',
+              level: 'debug',
+              msg: 'Log MessageA',
+              timestamp: '2023-01-05T10:12:32Z'
+            },
+            {
+              jobId: 'jobIdB',
+              level: 'debug',
+              msg: 'Log MessageB',
+              timestamp: '2023-01-05T10:12:32Z'
+            },
+            {
+              level: 'debug',
+              msg: 'Log Message',
+              timestamp: '2023-01-05T10:12:32Z'
+            }
+          ]
+        }
+      }
+    })
+
+    await sendKonnectorsLogs(client)
+    expect(mockFetchJSON).toHaveBeenCalledWith(
+      'POST',
+      '/konnectors/myslug/logs?job_id=jobIdA',
+      [
+        {
+          jobId: 'jobIdA',
+          level: 'debug',
+          msg: 'Log MessageA',
+          timestamp: '2023-01-05T10:12:32Z'
+        }
+      ]
+    )
+    expect(mockFetchJSON).toHaveBeenNthCalledWith(
+      2,
+      'POST',
+      '/konnectors/myslug/logs?job_id=jobIdB',
+      [
+        {
+          jobId: 'jobIdB',
+          level: 'debug',
+          msg: 'Log MessageB',
+          timestamp: '2023-01-05T10:12:32Z'
+        }
+      ]
+    )
+    expect(mockFetchJSON).toHaveBeenNthCalledWith(
+      3,
+      'POST',
+      '/konnectors/myslug/logs?job_id=',
+      [
+        {
+          level: 'debug',
+          msg: 'Log Message',
+          timestamp: '2023-01-05T10:12:32Z'
+        }
+      ]
+    )
+
+    expect(store.dispatch).toHaveBeenCalledWith({
+      payload: { number: 3, slug: 'myslug' },
+      type: 'konnectorLogs/removeLogs'
+    })
+  })
+
+  it('Should sort logs message by job id', async () => {
+    const cleanedLogs = [
+      {
+        jobId: 'jobIdA',
+        level: 'debug',
+        msg: 'Log MessageA',
+        timestamp: '2023-01-05T10:12:32Z'
+      },
```

Comment:
Can we have several logs for the same `jobIdA`? If yes, we should definitively test that

Reply:
J'ai ajouté un test qui envoie 3 logs avec le même jobIdA, et on vérifie qu'il n'y a qu'un seul appel à la stack avec les 3 logs.